frontend/public/components/filter-toolbar.tsx

```diff
@@ -242,69 +238,69 @@ const FilterToolbar_: React.FC<FilterToolbarProps & RouteComponentProps> = (prop
+                  <DataToolbarFilter
+                    key={key}
+                    chips={_.intersection(selectedRowFilters, filters[key]).map((item) => {
+                      return { key: item, node: filtersNameMap[item] };
+                    })}
+                    deleteChip={(filter, chip: DataToolbarChip) =>
+                      updateRowFilterSelected([chip.key])
+                    }
+                    categoryName={key}
+                    deleteChipGroup={() => clearAllRowFilter(key)}
                   >
-                    <FilterIcon className="span--icon__right-margin" />
-                    Filter
-                  </DropdownToggle>
-                }
-              />,
-            )}
-          </DataToolbarItem>
-        )}
-        <DataToolbarItem className="co-filter-search--full-width">
-          <DataToolbarFilter
-            deleteChipGroup={() => updateLabelFilter([])}
-            chips={[...labelFilters]}
-            deleteChip={(filter, chip: string) =>
-              updateLabelFilter(_.difference(labelFilters, [chip]))
-            }
-            categoryName="Label"
-          >
+                    {acc}
+                  </DataToolbarFilter>
+                ),
+                <Dropdown
+                  dropdownItems={dropdownItems}
+                  onSelect={onRowFilterSelect}
+                  isOpen={isOpen}
+                  toggle={
+                    <DropdownToggle
+                      data-test-id="filter-dropdown-toggle"
+                      onToggle={() => setOpen(!isOpen)}
+                      iconComponent={CaretDownIcon}
+                    >
+                      <FilterIcon className="span--icon__right-margin" />
+                      Filter
+                    </DropdownToggle>
+                  }
+                />,
+              )}
+            </DataToolbarItem>
+          )}
+          <DataToolbarItem className="co-filter-search--full-width">
             <DataToolbarFilter
-              chips={nameFilter && nameFilter.length > 0 ? [nameFilter] : []}
-              deleteChip={() => updateNameFilter('')}
-              categoryName="Name"
+              deleteChipGroup={() => updateLabelFilter([])}
+              chips={!hideLabelFilter ? [...labelFilters] : []}
+              deleteChip={(filter, chip: string) =>
+                updateLabelFilter(_.difference(labelFilters, [chip]))
+              }
+              categoryName="Label"
             >
-              <div className="pf-c-input-group">
-                {!hideLabelFilter && (
-                  <DropdownInternal
-                    items={FilterType}
-                    onChange={switchFilter}
-                    selectedKey={filterType}
-                    title={filterType}
-                  />
-                )}
-                {!hideNameFilter && (
+              <DataToolbarFilter
+                chips={nameFilter?.length > 0 ? [nameFilter] : []}
```

Comment:
I'd do this so I don't have think about what `undefined > 0` evaluates to. (I'm assuming it's false, but I shouldn't need to consider that :)

```suggestion
                chips={nameFilter?.length ? [nameFilter] : []}
```